modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -329,8 +332,40 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		to_chat(user, span_notice("Dead people can not be put into cryo."))
 		return
 
+// Allows admins to enable players to override SSD Time check.
+	if(allow_player_override)
+		if(tgui_alert(user, "Would you like to place [target] into [src]?", "Place into Cryopod?", list("Yes", "No")) != "No")
+			to_chat(user, span_danger("You put [target] into [src]. [target.p_theyre(capitalized = TRUE)] in the cryopod."))
+			log_admin("[key_name(user)] has put [key_name(target)] into a overridden stasis pod.")
+			message_admins("[key_name(user)] has put [key_name(target)] into a overridden stasis pod. [ADMIN_JMP(src)]")
+
+			add_fingerprint(target)
+
+			close_machine(target)
+			name = "[name] ([target.name])"
+
+// Allows players to cryo others. Checks if they have been AFK for 30 minutes.
 	if(target.key && user != target)
-		if(iscyborg(target))
+		if (target.getorgan(/obj/item/organ/brain) ) //Target the Brain
+			if(target.mind == null ) // Is the character empty / AI Controlled
+				if(target.lastclienttime + ssd_time MINUTES >= world.time)
+					to_chat(user, span_notice("You can't put [target] into [src]. They have not been asleep for [ssd_time] minutes, they have only been asleep for [round(((world.time - target.lastclienttime) / (1 MINUTES)),1)] minutes."))
+					log_admin("[key_name(user)] has attempted to put [key_name(target)] into a stasis pod.")
+					message_admins("[key_name(user)] has attempted to put [key_name(target)] into a stasis pod. [ADMIN_JMP(src)]")
+					return
+				else if(tgui_alert(user, "Would you like to place [target] into [src]?", "Place into Cryopod?", list("Yes", "No")) != "No")
```

Comment:
```suggestion
				else if(tgui_alert(user, "Would you like to place [target] into [src]?", "Place into Cryopod?", list("Yes", "No")) == "Yes")
```
In case they cancel the alert